_includes/footer.html

```diff
@@ -67,6 +67,13 @@ <h2>Resources</h2>
         </div>
       </div>
     </div>
+    <div class="privacy-policy">
+      <ul>
+        <li class="privacy-policy-links"><a href="assets/tos-oss-privacy-policy/fb-tos-privacy-policy.pdf" target="_blank">Terms</a></li>
+        <li class="privacy-policy-links">|</li>
+        <li class="privacy-policy-links"><a href="assets/tos-oss-privacy-policy/fb-oss-privacy-policy.pdf" target="_blank">Privacy</a></li>
```

Comment:
I don't think these links work on any other page besides the homepage. I think you'll have to update the URLs to  use `site.baseurl`: https://github.com/pytorch/pytorch.github.io/pull/352/files. Otherwise, this looks good. I would just ask if they want the links themselves to be white or match the gray color that the other footer links use.

Reply:
I'll ask Woo about the link color when I send the preview. I wasn't sure if the link should be gray or white to match the two top-level links(PyTorch and Resources)